chef/cookbooks/provisioner/templates/default/crowbar_join.suse.sh.erb

```diff
@@ -65,35 +65,19 @@ log_to() {
 
 get_state() {
   local output
-  local curlargs=(--connect-timeout 60 -s -L \
-      -H "Accept: application/json" -H "Content-Type: application/json" \
-      --max-time 240 --insecure --location)
-  [[ $CROWBAR_KEY ]] && curlargs+=(-u "$CROWBAR_KEY" --digest --anyauth)
-  # we expect something like {"value":"ready"}
-  output=$(curl "${curlargs[@]}" "http://$IP/dashboard/$1/attribute/state" | \
-      tr -d "{}\"'[:space:][:cntrl:]")
+  # we expect something like "status": "Ready"
+  output=$(crowbarctl node status --json --filter $1 | grep status | \
```

Comment:
I'm unsure about if /etc/crowbarrc is ensured to be present at this point, do you know?

Reply:
I believe at the time of crowbar_join calls `/etc/crowbarrc` is already installed (either the temporary one created by autoyast, or the real one created by `recipes/base.rb`